src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/AttachedCollections/Implementation/FrameworkReferenceAssemblyAttachedCollectionSourceProvider.cs

```diff
@@ -10,6 +10,7 @@
 
 namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.AttachedCollections.Implementation
 {
+    [AppliesTo(ProjectCapability.DependenciesTree)]
```

Comment:
We should consistently apply these attributes, we're mixing some from Utilities and some from CPS.

Reply:
They might end up with the same effective metadata, but sometimes they have different metadata for the same name, such as Order.

Reply:
What would you like to change specifically? `Microsoft.VisualStudio.Utilities.OrderAttribute` has different metadata than `Microsoft.VisualStudio.ProjectSystem.OrderAttribute`. We use ``Microsoft.VisualStudio.ProjectSystem.AppliesToAttribute` everywhere else, so it would be the odd one out. I can't see how it matters either way.

Reply:
It is a concidence that AppliesToAttribute produces to the same metadata as AppliesToProjectAttribute. If AppliesToProjectAttribute was changed to include new metadata, this usage would not be found.

There is no actual impact to the product at this very moment, but it's like mixing attributes between System.Composition and System.ComponentModel.Composition and just getting lucky that both work.

Reply:
NameAttribute is the same bucket.